<launch>
    <group ns="Drive">
        <node name="Drivetrain_Control" pkg="mavric" type="SparkCAN_Drive_Train.py" >
            <remap from="Drive_Train" to="Drive_Command" />
            <remap from="Steer_Train" to="Steer_Command" />
            <remap from="Pitch_Train" to="/Arm/ShoulderPitch" />
        </node>
        <node name="Drivetrain_Interface" pkg="mavric" type="Drive_Train_P.py" respawn="true" >
            <remap from="Drive_Train" to="Drive_Command" />
            <remap from="Steer_Train" to="Steer_Command" />
            <remap from="Steer_Cal" to="Cal_Command" />
            <param name="port" value="9002" />
        </node>
        <node name="Drivetrain_Heartbeat" pkg="mavric" type="DrivetrainHeartbeat.py" respawn="true" >
            <param name="timeout" value="1" />
            <remap from="Drive_Train" to="Drive_Command" />
        </node>
    </group>
</launch>